docs/Routes.md

```diff
@@ -42,6 +42,7 @@ They need to be in
   * `params`: validates the params.
   * `response`: filter and generate a schema for the response, setting a
     schema allows us to have 10-20% more throughput.
+* `exposeHeadRoute`: creates a sibling `HEAD` route from the current one. Only applies for `GET` routes. Defaults to the value of [`exposeHeadRoutes`](Server.md#exposeHeadRoutes) instance option.
```

Comment:
```suggestion
* `exposeHeadRoute`: creates a sibling `HEAD` route for any `GET` routes. Defaults to the value of [`exposeHeadRoutes`](Server.md#exposeHeadRoutes) instance option. If you want a custom `HEAD` handler without disabling this option, make sure to define it before the `GET` route.
```

Should also have a test for that case (there might already be, I haven't checked)

Reply:
Actually, it's not. Good catch, let me test this case. Thanks!

Reply:
Done in f636732
Please have a look 🙂